docs/tools/concepts.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Wallet Concepts
+hide_title: false
+---
+Vega Wallet is an app that lets you manage wallets and cryptographic key pairs. Key pairs are used to sign and send transactions in a secure way. 
+
+Read on to understand the basic concepts of wallets and keys, and how they work. 
+
+## Cryptographic keys
+Each cryptographic key pair includes a public key and a private key. Those keys are associated with an entity that needs to authenticate their identity, or sign and send transaction data. Data that is encrypted with the public key can only be decrypted with its corresponding private key.
```

Comment:
In our case, data are signed, not encrypted, and thus, it's the other way around. Signed with private key and verify with public key.